src/blog/non-backtracking-regexp.md

```diff
@@ -0,0 +1,184 @@
+﻿---
+title: 'An additional non-backtracking RegExp engine'
+author: 'Martin Bidlingmaier'

```

Comment:
Do you have a photo/avatar we can use? (See other blog posts)

Reply:
I don't have one at the moment that I would use for this. Is it strictly necessary?

Reply:
Ideally it would be good, but not strictly necessary, no.